include/deal.II/fe/fe_poly_tensor.h

```diff
@@ -224,6 +224,122 @@ class FE_PolyTensor : public FiniteElement<dim, spacedim>
   bool
   single_mapping_kind() const;
 
+  /**
+   * The vector of tables
+   * <code>adjust_quad_dof_index_for_face_orientation_table</code> contains
+   * offsets for local face dofs and is being filled with zeros in fe.cc. We
+   * need to fill it with the correct values in case of non-standard, flipped
+   * (rotated by +180 degrees) or rotated (rotated by +90 degrees) faces. These
+   * are given in the form three flags (face_orientation, face_flip,
+   * face_rotation), see the documentation in GeometryInfo<dim> and
+   * this @ref GlossFaceOrientation "glossary entry on face orientation".
+   *
+   * <h3>Example: Raviart-Thomas Elements of order 2 (tensor polynomial degree
+   * 4)</h3>
+   *
+   * The dofs on a face are connected to a <code>n</code>-by-<code>n</code>
+   * matrix where here <code>n=3</code>. In our example we can imagine the
+   * following dofs on a quad (face):
+   *
+   * @verbatim
+   *  ___________
+   * |           |
+   * |  6  7  8  |
+   * |           |
+   * |  3  4  5  |
+   * |           |
+   * |  0  1  2  |
+   * |___________|
+   *@endverbatim
+   *
+   * We have for a local <code>face_dof_index=i+n*j</code> with index
+   * <code>i</code> in x-direction and index <code>j</code> in y-direction
+   * running from 0 to <code>n-1</code>.  To extract <code>i</code> and
+   * <code>j</code> we can use <code>i = face_dof_index % n</code> and <code>j =
+   * dof_index / n</code> (integer division). The indices <code>i</code> and
+   * <code>j</code> can then be used to compute the offset.
```

Comment:
This may have been a bit confusing. The comment underlines how this can be achieved for the RT-element but not how to do that for other elements. For other H(div) elements such as BDM such a technique may not work. The only message here is: "There is a permutation and a sign change. How to find it depends on the element." 

I rephrased the documentation in the base class and now it hopefully is clearer to the reader.